MiniSim/MiniSim.swift

```diff
@@ -110,8 +117,13 @@ class MiniSim: NSObject {
     private func setMenuImage() {
         if let button = statusItem.button {
             button.toolTip = "MiniSim"
-            let itemImage = NSImage(named: "menu_icon")
-            itemImage?.size = NSSize(width: 9, height: 16)
+            let itemImage = NSImage(named: UserDefaults.standard.menuImage)
+            if (!menuImagesObserved.contains(UserDefaults.standard.menuImage)) {
+                itemImage?.size = NSSize(
```

Comment:
You can set the size without this if check as we have the `?.size` optional check